spec/acceptance/getpwnam_spec.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+describe 'function stdlib::getpwnam' do
```

Comment:
Better make the whole test conditional?

```suggestion
describe 'function stdlib::getpwnam', unless os[:family] == 'windows' do
```